infra/recipes/terraform/azure/environments/sandbox/README.md

```diff
@@ -34,6 +34,20 @@ You could set the environment variables for variables to stored terraform backen
 `cp backend.tfvars.template backend.tfvars`.
 2. Edit the backend.tfvars to configure the necessary TF backend state.
 
+## Docker setup
+
+To be able to build the Tables and Housetables service images, you will need to ensure Docker is running on your local machine. This is typically done by opening Docker Desktop. You can check for this by running a command like `docker info` in the terminal.
+
+Sometimes, the images may stop building or uploading to the ACR. This is likely because docker has run out of local memory. To fix this, run `docker system prune --volumes` locally.
```

Comment:
Can you add step by step instruction to build and upload to ACR?

Reply:
Yes, it is mostly done automatically by running `terraform apply`, but I will include that